test/extended/tnf/recovery.go

```diff
@@ -0,0 +1,199 @@
+package tnf
+
+import (
+	"context"
+	"fmt"
+	"math/rand"
+	"slices"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+	v1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/origin/test/extended/etcd/helpers"
+	exutil "github.com/openshift/origin/test/extended/util"
+	"github.com/pkg/errors"
+	"go.etcd.io/etcd/api/v3/etcdserverpb"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = g.Describe("[sig-node][apigroup:config.openshift.io] Two Nodes OCP with fencing recovery", func() {
```

Comment:
Can you please add the annotation `[OCPFeatureGate:DualReplica]` to the test names to allow the feature gate to be captured

Reply:
Sure :+1: